src/app/services/http-services/Membership.js

```diff
@@ -0,0 +1,66 @@
+angular.module('app')
+  .factory('Membership', ($q, $http) => {
+    let baseUrl = 'http://localhost:3000/api/membership'
```

Comment:
I dont think you want to do the full url. `http://localhost:3000` will be removed when its hosted somewhere. Use a relative path.